src/app/Core/calculateRowsWithPercent.service.ts

```diff
@@ -0,0 +1,43 @@
+function getSumOfIsolates(objList: Record<string, string>[]): number {
+    let sum = 0;
+    objList.forEach((obj) => {
+        const k = Object.keys(obj);
+        k.forEach((element) => {
+            if (element !== "name") {
+                sum += (obj[element] as unknown) as number;

```

Comment:
reject: use something like:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
telling typescript to ignore the type doesn't prevent your app from crashing if something goes wrong.